### PR TITLE
Enable peribolos-syncer job for all repos

### LINKED ISSUE #1343: Run peribolos-syncer jobs on all repositories
**Motivation**

Peribolos-syncer job is run only for evolution repository. It should be enabled for all repositories.

**Feature**

Run peribolos-syncer jobs on all repositories

**Alternatives**

NA

**Additional context**

NA

### PR DESCRIPTION
**Pull request by maxgio92**
Fixes #1343

### CODE REVIEW

File: config/jobs/update-github-teams/peribolos-syncer-charts.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - name: peribolos-syncer-post-submit
+    branches:
+    - ^main$
+    decorate: true
+    max_concurrency: 1
+    skip_report: false
+    run_if_changed: 'OWNERS$'
+    spec:
+      containers:
+      - image: gcr.io/maxgio92/peribolos-syncer:0.2.0
```
Comment (maxgio92) [inline config/jobs/update-github-teams/peribolos-syncer-charts.yaml:16]: Before using the `falcosecurity` package, we need https://github.com/falcosecurity/peribolos-syncer/pull/11.

**Comment by maxgio92** [comment]
@FedeDP ready for review

File: config/jobs/update-github-teams/peribolos-syncer-libs.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        args:
+        - sync
+        - github
+        - --org=falcosecurity
+        - --team=libs-maintainers
+        - --peribolos-config-path=config/org.yaml
+        - --peribolos-config-repository=test-infra
+        - --peribolos-config-git-ref=master
+        - --owners-repository=libs
+        - --owners-git-ref=main
```
Comment (FedeDP) [inline config/jobs/update-github-teams/peribolos-syncer-libs.yaml:22]: Shouldn't this be `master` for eg:  driverkit, libs, falco... repositories?

Comment (maxgio92) [reply config/jobs/update-github-teams/peribolos-syncer-libs.yaml:22]: You're right, good catch! Thanks

Comment (maxgio92) [reply config/jobs/update-github-teams/peribolos-syncer-libs.yaml]: @FedeDP it should be aligned now.

Plus, I've removed configurations for the libsinsp and the libscap repos.

File: config/jobs/update-github-teams/peribolos-syncer-charts.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        args:
+        - sync
+        - github
+        - --org=falcosecurity
+        - --team=charts-maintainers
+        - --peribolos-config-path=config/org.yaml
+        - --peribolos-config-repository=test-infra
+        - --peribolos-config-git-ref=master
+        - --owners-repository=charts
+        - --owners-git-ref=main
```
Comment (FedeDP) [inline config/jobs/update-github-teams/peribolos-syncer-charts.yaml:26]: `master` : https://github.com/falcosecurity/charts

**Comment by FedeDP** [comment]
Looking at https://github.com/falcosecurity/test-infra/blob/master/config/config.yaml, we miss:
* event-generator
* dbg-go
* cncf-green-review-testing
* client-py
* client-rs
* falco-aws-terraform
* falco-exporter
* falco-playground
* flycheck-falco-rules
* k8s-metacollector
* kilt
* libs-sdk-go
* pdig
* plugin-sdk-go
* plugin-sdk-cpp
* syscalls-bumper

Some of them might be archived thus there is no need to add them (i guess?), even if, since they are listed in config yaml, i'd definitely add them here too.

File: config/jobs/update-github-teams/peribolos-syncer-charts.yaml (hunk 3 above)
Comment (maxgio92) [reply config/jobs/update-github-teams/peribolos-syncer-charts.yaml]: Applied thanks @FedeDP

**Comment by maxgio92** [comment]
Hey @FedeDP, thanks and sorry for the big delay. I've added the conf for the mentioned repos, except for the archive repositories:
- client-py
- client-rs
- kilt
- pdig

WDYT?

**Review by FedeDP — APPROVED** [review]
/approve
Thanks, now this LGTM!

**Comment by FedeDP** [comment]
Feel free to unhold whenever you wish ;)

**Pull request merged**